### PR TITLE
Switch to neutral, flat icons for voting

### PR DESCRIPTION
**Pull request by schalkneethling**
Switch to flat icons with a neutral colour scheme that better fits with the overall design. @jwhitlock r?

### CODE REVIEW

File: jinja2/includes/icons/emojis/thumbs-down.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...ox="0 0 914.286 1024" class="icon icon-thumbs-down"><path class="path0" stroke="inherit" d="M146.286 329.143c0 20-16.571 36.571-36.571 36.571-20.571 0-36.571-16.571-36.571-36.571 0-20.571 16-36.571 36.571-36.571 20 0 36.571 16 36.571 36.571zm91.428 292.571V256c0-20-16.571-36.571-36.571-36.571H36.572C16.572 219.429.001 236 .001 256v365.714c0 20 16.571 36.571 36.571 36.571h164.571c20 0 36.571-16.571 36.571-36.571zm645.143-85.143c19.429 21.714 31.429 54.857 31.429 85.143-.571 59.429-50.286 109.714-109.714 109.714H646.286c4.571 18.286 10.286 24 16.571 36.571 14.857 29.714 32 62.857 32 109.714 0 44 0 146.286-128 146.286-9.714 0-18.857-4-25.714-10.857-24.571-24-31.429-59.429-37.714-93.143-6.857-33.143-13.143-67.429-35.429-89.714-17.714-17.714-37.143-42.286-57.714-68.571-25.143-33.143-80-101.143-101.143-102.857-18.857-1.714-34.857-17.714-34.857-36.571V256c0-20 17.143-36 36.571-36.571 20-.571 54.286-12.571 90.286-25.143 61.714-21.143 138.857-48 220.571-48h73.714c50.286.571 88 15.429 112.571 44.571 21.714 25.714 31.429 60.571 28 103.429C850.285 308 861.142 326.857 866.856 348c6.286 22.857 6.286 45.714 0 66.857 17.143 22.857 25.714 49.714 24.571 78.286 0 8-2.286 25.143-8.571 43.429z"/></svg>
```
Comment (jwhitlock) [inline jinja2/includes/icons/emojis/thumbs-down.svg]: Github says this SVG is invalid - does it work for you?

Also, does it need "aria-hidden"? Or should that be removed from ``thumbs-up.svg``?

File: jinja2/includes/icons/emojis/thumbs-up.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...6 1024" class="icon icon-thumbs-up" aria-hidden="true"><path class="path0" stroke="inherit" d="M146.286 768c0-20-16.571-36.571-36.571-36.571-20.571 0-36.571 16.571-36.571 36.571 0 20.571 16 36.571 36.571 36.571 20 0 36.571-16 36.571-36.571zm91.428-292.571v365.714c0 20-16.571 36.571-36.571 36.571H36.572c-20 0-36.571-16.571-36.571-36.571V475.429c0-20 16.571-36.571 36.571-36.571h164.571c20 0 36.571 16.571 36.571 36.571zm676.572 0c0 30.286-12 62.857-31.429 85.143 6.286 18.286 8.571 35.429 8.571 43.429 1.143 28.571-7.429 55.429-24.571 78.286 6.286 21.143 6.286 44 0 66.857-5.714 21.143-16.571 40-30.857 53.714 3.429 42.857-6.286 77.714-28 103.429-24.571 29.143-62.286 44-112.571 44.571h-73.714c-81.714 0-158.857-26.857-220.571-48-36-12.571-70.286-24.571-90.286-25.143-19.429-.571-36.571-16.571-36.571-36.571V474.858c0-18.857 16-34.857 34.857-36.571 21.143-1.714 76-69.714 101.143-102.857 20.571-26.286 40-50.857 57.714-68.571 22.286-22.286 28.571-56.571 35.429-89.714 6.286-33.714 13.143-69.143 37.714-93.143 6.857-6.857 16-10.857 25.714-10.857 128 0 128 102.286 128 146.286 0 46.857-16.571 80-32 109.714-6.286 12.571-12 18.286-16.571 36.571h158.286c59.429 0 109.714 50.286 109.714 109.714z"/></svg>
```
Comment (jwhitlock) [inline jinja2/includes/icons/emojis/thumbs-up.svg]: GitHub says this is an invalid SVG. And maybe ``aria-hidden`` should be removed, or added to ``thumbs-down``.

File: kuma/static/styles/components/contribution/page.scss [hunk 3]
```diff
@@ -108,14 +108,19 @@
...
+                    color: #fff;
                 }
 
                 .icon {
                     pointer-events: none;
                 }
             }
+
+            .icon-thumbs-down {
+                transform: scaleX(-1);
```
Comment (jwhitlock) [inline kuma/static/styles/components/contribution/page.scss:122]: I'm sure this vertical flip will make more sense when I can see the SVG.

**Review by jwhitlock — CHANGES_REQUESTED** [review]
@schalkneethling can you see if the SVG is valid for you? I'm not aware of other SVG files that GitHub has issues with.

File: jinja2/includes/icons/emojis/thumbs-down.svg (hunk 1 above)
Comment (schalkneethling) [reply jinja2/includes/icons/emojis/thumbs-down.svg]: They worked for me locally :/ I also have ru them through SVGOMG and it also did not complain. I missed the `aria-hidden`, my bad.

**Comment by schalkneethling** [comment]
@jwhitlock 	updated, see if you have better luck this time round. Thanks!

**Comment by jwhitlock** [comment]
Nope, still invalid. I think the ``<svg>`` needs a ``xmlns`` element:

```
<svg xmlns="http://www.w3.org/2000/svg" ...
```

**Comment by schalkneethling** [comment]
> Nope, still invalid. I think the <svg> needs a xmlns element:

Indeed it does, strange that nothing complained on my side, hmmmm.

**Comment by jwhitlock** [comment]
This seems accurate: https://stackoverflow.com/a/34249810/10612

>  The xmlns="http://www.w3.org/2000/svg" attribute is:
>  * Required for image/svg+xml files.
>  * Optional for inlined <svg>

When we update the asset pipeline, we'll commit regular, un-optimized SVG files, and drop the ``xmlns`` tag for inline-content SVG. For now, I'd prefer to keep it for SVG files that happen to be used inline.

**Review by jwhitlock — APPROVED** [review]
This looks good locally.

We should switch to "Yes" / "No", since 👍 does not universally mean "Yes" - https://www.ethos3.com/2014/07/dont-give-a-thumbs-up-in-russia-and-other-hand-gesture-facts/

https://health.onehowto.com/article/the-meaning-of-hand-gestures-in-different-countries-1326.html#anchor_0

**Pull request merged**